spec/rails_helper.rb

```diff
@@ -61,3 +61,10 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
why not to move this to a support file? You already have the support for that enabled on line spec/rails_helper.rb:23